exporters/trace/ocagent/src/main/java/io/opencensus/exporter/trace/ocagent/TraceProtoUtils.java

```diff
@@ -378,5 +375,26 @@ public static TraceParams fromTraceConfigProto(
     return builder.build();
   }
 
+  // Creates a CurrentLibraryConfig message with the given Node and current TraceParams.
+  static CurrentLibraryConfig getCurrentLibraryConfig(
+      @javax.annotation.Nullable Node node, io.opencensus.trace.config.TraceConfig traceConfig) {
```

Comment:
Probably better to return the TraceConfig proto here and avoid adding the only first time send the node logic in this util class.

Reply:
Make sense, done.